tests/testutil.hpp

```diff
@@ -32,7 +32,7 @@
 
 #include "../include/zmq.h"
 #include "../src/stdint.hpp"
-#include "platform.hpp"
+#include "../src/platform.hpp"
```

Comment:
For sure. Sorry, I did not realize. I've broken this multiple times when
building/testing individual cases. It might be worth wrapping this include
in a CMAKE-specific macro so that it's clear. Otherwise, maybe a comment...

On Mon, Jul 20, 2015 at 1:46 PM, Richard Newton <notifications@github.com>
wrote:

> In tests/testutil.hpp
> <https://github.com/zeromq/libzmq/pull/1476#discussion_r34986886>:
>
> > @@ -32,7 +32,7 @@
> >
> >  #include "../include/zmq.h"
> >  #include "../src/stdint.hpp"
> > -#include "platform.hpp"
> > +#include "../src/platform.hpp"
>
> Changing the platform include like this always breaks the cmake build
> where the plaform.hpp is generated out of source.
>
> Can we change it back?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zeromq/libzmq/pull/1476/files#r34986886>.
>